tsconfig.json

```
@@ -1,20 +1,26 @@
 {
 	"compilerOptions": {
-		"target": "es5",
+		"target": "esnext",
```

Since this file was generated by Next.js itself, we should keep the target to `es5` for now, just to be safe.
```suggestion
		"target": "es5",
```